 2:55:09.05  Test starded without threads
00:55:09.575 Python Module Path 'Scripts/Gateway/SystemAreas/OrderProcessing/PythonModules' added.
00:55:09.575 Python Module Path 'ScriptKit/TroubleMaker' added.
00:55:09.591 Python Module Path 'Scripts/Gateway/PythonModules' added.
00:55:09.591 Python Module Path 'ScriptKit' added.
00:55:10.028 2020-02-19 Root suite 'Scripts/Gateway/SystemAreas/OrderProcessing/RiskEvaluation/Suites/SynthSpread_VariableTickSize' full execution.
00:55:10.043 Suite '.'. [8 test cases, 0 sub-suites]
Traceback (most recent call last):
  File "C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py", line 33, in invoke_script
    exec(code, globals())
  File "Scripts/Gateway/SystemAreas/OrderProcessing/RiskEvaluation/Suites/SynthSpread_VariableTickSize\test_suite.su.py", line 77, in <module>
    generator.Open()
  File "C:\xpit.com\applications\ScriptFramework\ScriptKit\IHEGenerator.py", line 44, in Open
    self.__generator.Open(self.__name)
  File "C:\xpit.com\applications\ScriptFramework\ScriptKit\ModuleAgent.py", line 8898, in Open
    return _ModuleAgent.IIHEGenerator_Open(self, name)
RuntimeError: Failed to resolve interface  (from Interface): interface not found.
Available interfaces:
name=\DEVICE\TCPIP_{CFF7A875-51FB-48C1-9B78-ABA35ACC4BDF}, ip=172.26.65.55, network=172.26.65.0, up=1
name=\DEVICE\TCPIP_{1F777394-0B42-11E3-80AD-806E6F6E6963}, ip=127.0.0.1, network=127.0.0.0, up=1


00:55:19.122 Suite '.' setup failed, child test cases and sub-suites will be skipped.
00:55:19.731 Root suite 'Scripts/Gateway/SystemAreas/OrderProcessing/RiskEvaluation/Suites/SynthSpread_VariableTickSize' execution finished. It took 0:00:09.687658.
00:55:19.731 Test cases results: 8 total, 0 succeeded, 0 failed, 8 skipped.
